src/applications/simple-forms/form-upload/containers/ConfirmationPage.jsx

```diff
@@ -4,12 +4,9 @@ import { useSelector } from 'react-redux';
 import { ConfirmationPageView } from '../../shared/components/ConfirmationPageView';
 
 const content = {
-  headlineText: 'Your form has been submitted and is pending processing',
+  headlineText: "You've submitted your form",
```

Comment:
Super tiny nit: I see a `'` here and usually I feel like we have this character, `’`, for apostrophes.

Reply:
Good catch, fixed! 👍